KarhooUISDK/Screens/QuoteList/QuoteListFilters/Filters/LuggageCapacityFilter.swift

```diff
@@ -11,10 +11,11 @@ import KarhooSDK
 
 extension QuoteListFilters {
     struct LuggageCapacityModel: QuoteListNumericFilter {
+        static let defaultValue: Int = 0
```

Comment:
Do you need this static value? If yes, then I suggest to remove 
`static let defaultPassengersCount = 1
    static let defaultLuggagesCount = 0 `
from `QuoteListFilters ` enum and use:
`var defaultValue: Int { Self.defaultValue } ` it this struct and in 'PassengerCapacityModel'
In QuoteFilterHandle you can use 'PassengerCapacityModel.defaultValue' while computing 'numberOfPassangers'

Reply:
No, it's leftover. I did decide to go with `QuoteListFilters. defaultLuggagesCount` etc To make those default values not-linked to specific structure.